contracts/common/SafeERC20.sol

```diff
@@ -65,4 +91,25 @@ library SafeERC20 {
         GeneralERC20(_token).approve(_spender, _amount);
         return checkSuccess();
     }
+
+    function staticBalanceOf(ERC20 _token, address _owner) internal view returns (uint256 tokenBalance) {
+        bytes memory balanceOfCallData = abi.encodeWithSelector(
+            ERC20(_token).balanceOf.selector,
+            _owner
+        );
+        bool success;
+        (success, tokenBalance) = staticInvoke(_token, balanceOfCallData);
+        require(success, ERROR_TOKEN_BALANCE_REVERTED);
```

Comment:
I would do an explicit return

Reply:
Main reason I tend to favour the implicit return is finding out that it's a little bit cheaper than an explicit return.

Although in this case, it's probably worth being explicit since the value's set as part of a tuple.